### PR TITLE
ARC-253 analytics to add

### PR DESCRIPTION
**Pull request by elehma4**

### CODE REVIEW

File: src/components/dashboard/SignSettings.tsx [hunk 1]
```diff
@@ -37,12 +43,22 @@ export default function SignSettings() {
...
   };
 
-  const handleAllowanceChange = async (e) => {
+  const handleBlur = async (e) => {
     const newAllowance = Number(e.target.value);
-    setSignatureAllowance(newAllowance);
+    if (newAllowance !== signatureAllowance) {
+      trackEvent(EventType.SEND_ALLOWANCE_CHANGE, {
+        Before: signatureAllowance,
+        After: newAllowance
```
Comment (nicholaswma) [inline src/components/dashboard/SignSettings.tsx]: before and after should be lowercase to follow camelcasing

File: src/routes/popup/send/confirm.tsx [hunk 2]
```diff
@@ -334,6 +335,11 @@ export default function Confirm({ tokenID, qty }: Props) {
             content: browser.i18n.getMessage("sent_tx"),
             duration: 2000
           });
+          trackEvent(EventType.TX_SENT, {
+            contact: contact ? true : false,
+            amount: transactionAmount,
+            fee: networkFee
+          });
```
Comment (nicholaswma) [inline src/routes/popup/send/confirm.tsx:342]: we'll need to trackEvent on fallback case as well down on line ~400.
transaction amount from what i remember could be AR or other token types, If we're only monitoring AR transfers, we could have a check for `token === "AR" ? transactionAmount : 0`

**Review by nicholaswma — COMMENTED** [review]
Added two minor points, but should be good after those!

**Review by 7i7o — APPROVED** [review]
utACK 🚀

**Review by nicholaswma — APPROVED** [review]

**Pull request merged**